samples/java/tutorial_code/features/feature_description/SURFMatchingDemo.java

```diff
@@ -2,8 +2,8 @@
 import org.opencv.core.Mat;
 import org.opencv.core.MatOfDMatch;
 import org.opencv.core.MatOfKeyPoint;
-import org.opencv.features2d.DescriptorMatcher;
-import org.opencv.features2d.Features2d;
+import org.opencv.features.DescriptorMatcher;
+import org.opencv.features.Features;
```

Comment:
The same.

Reply:
I am not familiar with the java binding, and changing back to `Features2d` leads to compile errors.